lib/smart_answer_flows/student-finance-forms.rb

```diff
@@ -73,8 +73,12 @@ def define
       end
 
       multiple_choice :what_year_full_time? do
-        option 'year-1617'
-        option 'year-1516'
+        question_value = self
+        precalculate :setup_options do
+          question_value.option "year-1718" if type_of_student == "uk-full-time"
+          question_value.option "year-1617"
+          question_value.option "year-1516" if type_of_student == "eu-full-time"
+        end
 
```

Comment:
I have split the commits ea0fb4ca66563f946e5558437765401422144ccb, f0026fc17fdd452438e601d89fbd5ccc66f5f660